signature/jws/signer_test.go

```diff
@@ -176,3 +177,35 @@ func generateKeyCertPair() (*rsa.PrivateKey, *x509.Certificate, error) {
 	}
 	return key, cert, nil
 }
+
+func TestNewSignerFromCertificate(t *testing.T) {
+	key, cert, err := generateKeyCertPair()
+	if err != nil {
+		t.Fatalf("generateKeyCertPair() error = %v", err)
+	}
+	tlscert := tls.Certificate{
+		Certificate:                  [][]byte{cert.Raw},
+		PrivateKey:                   key,
+		SupportedSignatureAlgorithms: []tls.SignatureScheme{tls.ECDSAWithP521AndSHA512},
+	}
+	s, err := NewSignerFromCertificate(tlscert)
```

Comment:
Consider testing the case where `len(tlscert.Certificate) == 0` to ensure the error case is handled correctly

Reply:
Done!